content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -441,6 +441,12 @@ deprecating and removing a metric are as follows:
    * **STABLE: 3 releases or 9 months (whichever is longer)**
    * **BETA: 1 releases or 4 months (whichever is longer)**
    * **ALPHA: 0 releases**
+   
+It is permissible, in certain cases, for metrics to skip stability levels in the process
```

Comment:
The doc on the website is specific to the deprecation policies, maybe we should rather move that to https://github.com/kubernetes/community/blob/53ccd27e43731fe76bff3a00c602d53eb5ed55ae/contributors/devel/sig-instrumentation/metric-stability.md and update that doc while we are at it?

Reply:
Let's update it in both spots. I'll open up another PR.